doc/03-Configuration.md

```diff
@@ -0,0 +1,115 @@
+# Configuration
+
+The configuration is stored in `/etc/icinga-notifications/config.yml`.
+See [config.example.yml](../config.example.yml) for an example configuration.
+
+Please note that this configuration file only contains the essentials.
+Detailed configuration as, i.e., the users, schedule, or channels, will be configured through Icinga Notifications Web.
+
+## Top Level Configuration
+
+### Listener Configuration
+
+The HTTP listener can be used both for submission and for debugging purposes.
+
+| Option         | Description                                                          |
+|----------------|----------------------------------------------------------------------|
+| listen         | Address to be bound to.                                              |
+| debug-password | Password expected via HTTP Basic Authentication for debug endpoints. |
+
+### Icinga Web 2
+
+The `icingaweb2-url` is expected to point to the base directory of your Icinga Web 2 installation,
+i.e., `https://example.com/icingaweb2/`, to be used for URL creation.
+
+### Channel Plugin Directory
+
+All available Icinga Notification Channel Plugins should resident in the `channel-plugin-dir`.
+For a package install, this directory should be `/usr/libexec/icinga-notifications/channels`.
```

Comment:
For a package install, it should not be necessary to have this option in the config file at all.

Reply:
Fair enough. I have rephrased this section to mention that this configuration option shouldn't be needed for package installs. However, I dislike hiding some information which might be useful to some users in some special setups.